app/models/application.rb

```diff
@@ -0,0 +1,7 @@
+class Application < ApplicationRecord
+
+  validates_presence_of :name, :address, :city, :state, :zip, :phone_number, :description
+  has_many :pet_applications
+  has_many :pets, through: :pet_applications
+
+end
```

Comment:
application table create with many to many relationship with pets through join table of pet_applications